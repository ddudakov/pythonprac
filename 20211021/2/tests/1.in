:sin x sin(x)
sin 1
:decorate s "--<<{}>>--".format(s)
decorate "ЖЖЖ"
:q_func "qwe"
q_func
quit "{}:{}"
