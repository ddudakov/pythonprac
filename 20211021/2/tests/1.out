0.8414709848078965
--<<ЖЖЖ>>--
qwe
4:7
